src/chainer.test.js

```diff
@@ -1 +1,29 @@
-const { chainer } = require('./chainer');
+"use strict";
+
+describe(`chainer`, () => {
+  const { chainer } = require("./chainer");
+
+  it(`should correctly chain and execute functions with arguments`, () => {
+    const f1 = jest.fn((x) => x * 2);
+    const f2 = jest.fn((x) => x + 2);
+    const f3 = jest.fn((x) => Math.pow(x, 2));
+
+    chainer(f1, f2, f3);
```

Comment:
The `chainer` function should be called with an array of functions, e.g., `chainer([f1, f2, f3])`. This line currently passes the functions as separate arguments, which is incorrect.